docs/installing-solidity.rst

```diff
@@ -273,6 +273,10 @@ Command-Line Build
 
 **Be sure to install External Dependencies (see above) before build.**
 
+If you want to perform a source build, please only use solidity.tar.gz in `release page`_ and not the zip provided by github directly.
```

Comment:
This comment only applies to making a release build, but it should be unrelated to the problem you encountered.